### PR TITLE
Move visibility to resources level

### PR DESCRIPTION
**Pull request by avdata99**
Related to [UNHCR#581](https://github.com/okfn/ckanext-unhcr/issues/581)
Requered for [UNHCR#583](https://github.com/okfn/ckanext-unhcr/pull/583)

Ensure `visibility` is defined for resources

### CODE REVIEW

**Comment by chris48s** [comment]
Hmm. This asks an interesting question (which I do not have an answer to yet :slightly_smiling_face: ).

At the moment when a user uploads through the frontend we ask the user for "visibility" at the dataset level on the import form and we just apply that answer to the dataset.

Now we are moving it to the resource level, we need to set visibility to something for each resource. We are uploading two resources but we are not showing the user any resource form, so we have to make a decision..

- One thing we can do is just assume "restricted", which is what you've done here basically. It is the most conservative default. If the user wants to change it, they can edit the resources later.
- Another choice we could make here would be to assume "internally available". This might seem like an unsafe default, but the DDI XML/RDF are meta-data (there would be nothing personally identifiable in here). Also, a lot of the info from the XML ends up exposed in the dataset metadata anyway which can be seen by any user.
- The final option would be to keep on asking for this field on the import form and then apply the answer to both resources instead of the dataset.

@amercader - do you have any views on this? We could run it by the client on Thursday if not..

**Comment by amercader** [comment]
FWIW I like the third option, but we can confirm this with the client:

> The final option would be to keep on asking for this field on the import form and then apply the answer to both resources instead of the dataset.

**Comment by chris48s** [comment]
I've stuck it on the list to ask in the meeting tomorrow :+1:

**Comment by chris48s** [comment]
We followed up with the client today. The decision is (as @amercader suggests) option 3:

> keep on asking for this field on the import form and then apply the answer to both uploaded resources instead of the dataset.

Can you have a look at that please @avdata99

File: ckanext/ddi/blueprints.py [hunk 1]
```diff
@@ -140,7 +140,6 @@ def get(self, package_type, data=None, errors=None, error_summary=None):
                 u'form_snippet': form_snippet,
                 u'dataset_type': package_type,
                 u'resources_json': json.dumps(data.get('resources', [])),
-                u'form_snippet': form_snippet,
```
Comment (avdata99) [inline ckanext/ddi/blueprints.py:143]: just a duplicated key

**Comment by avdata99** [comment]
This ready for review @chris48s 
https://github.com/okfn/docker-ckan-unhcr-aws/pull/46 was added to update DDI at docker repo (require to tag as `v2.0.2` if merged).

**Review by chris48s — APPROVED** [review]
:+1: think this is good to go now but lets co-ordinate merging/tagging it with https://github.com/okfn/ckanext-unhcr/pull/583

**Review by amercader — APPROVED** [review]

**Pull request merged**